src/commands.ts

```diff
@@ -1560,17 +1560,14 @@ const isCommentingAllowed = (type: NodeType) =>
   type === schema.nodes.table_element ||
   type === schema.nodes.embed ||
   type === schema.nodes.affiliations ||
-  type === schema.nodes.contributors
+  type === schema.nodes.contributors ||
+  type === schema.nodes.image_element
 
 export const addNodeComment = (
   node: ManuscriptNode,
   state: ManuscriptEditorState,
   dispatch?: Dispatch
 ) => {
-  if (!isCommentingAllowed(node.type)) {
```

Comment:
Maybe it's better to keep it also here just in case this `addNodeComment` is called from somewhere out the context menu, what do you think?

Reply:
updated